types/hapi/index.d.ts

```diff
@@ -2535,6 +2546,13 @@ export interface ServerInfo {
     uri: string;
 }
 
+/**
+ * Empty interface to allow for custom augmentation.
+ */
+/* tslint:disable-next-line:no-empty-interface */
+export interface ServerInjectOptionsApp {
```

Comment:
From what I can tell these should be the same as `RouteOptionsApp `.

Reply:
The description says `sets the initial value of request.app, defaults to {}.`.  The type of request.app in our typings now is currently ApplicationState.  Would that be the correct type?

Reply:
I changed it as I have it above, as I think it is correct.  If I'm misreading, let me know and I can change it to how you suggested.